gulp_tasks/locales.js

```diff
@@ -1,12 +1,50 @@
 const gulp = require('gulp');
-const rename = require('gulp-rename');
+const fs = require('fs');
+const path = require('path');
+const merge = require('merge-stream');
+const merge_json = require('gulp-merge-json');
 
-gulp.task('locales', locales);
+const conf = require('../conf/gulp.conf');
+const localesPath = 'bower_components/impac-angular/dist/locales/';
+
+// Return a list of locales file from a folder
+function getLocales(dir) {
+  return fs.readdirSync(dir)
+    .filter(function(file) {
+      return file.substr(-5) === '.json';
+    });
+}
+
+// Append their locale to our locale
+function mergeLocales(our, their) {
+  return gulp.src([
+    conf.path.src('locales/', our),
+    path.join(localesPath, their)
+  ], {allowEmpty: true})
+    .pipe(merge_json({fileName: our}))
+    .pipe(gulp.dest(conf.path.tmp('locales')))
+}
 
+// Get the locales from impac-angular and append them to our locales
+// Only in dev are we are generating the locales from YAML for prod build
 function locales() {
-  return gulp.src('bower_components/impac-angular/dist/locales/*')
-    .pipe(rename(function (filename) {
-      filename.basename = filename.basename.slice(0, 2);
-    }))
-    .pipe(gulp.dest('src/locales/impac'))
+  var locales = getLocales(localesPath);
+
+  // Executes the function once per file and returns the async stream
+  // This will append 'en-AU.json' to 'en-AU.json'
```

Comment:
`// This will append 'some-dir/en-AU.json' to 'src/locales/en-AU.json'`

Reply:
Done